nupic/test/abstract_temporal_memory_test.py

```diff
@@ -24,13 +24,19 @@
 from nupic.data.generators.sequence_machine import SequenceMachine
 from nupic.research.monitor_mixin.temporal_memory_monitor_mixin import (
   TemporalMemoryMonitorMixin)
-from nupic.research.temporal_memory import TemporalMemory
+
+# Uncomment one line below to run tests with C++ or Py TM implementation instead
+from nupic.bindings.algorithms import TemporalMemory
+#from nupic.research.temporal_memory import TemporalMemory
```

Comment:
With a local develop build of nupic, this should be the only place to swap between C++ and Python implementations for the integration tests. But the comment above regarding the compute calls issue shows another change required in the TM mixin code to swap between implementations.